tests/common/unittest_common.cc

```diff
@@ -686,6 +686,7 @@ TEST (common_string_util, replace_str_02)
   g_free (result);
 }
 
+#ifndef __TIZEN__
```

Comment:
You don't need this.
In Tizen, unittest is supposed to be executed in GBS-nnstreamer, which is supposd to have NO nnstreamer.ini at /etc.

Reply:
Right. updated.